externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala

```diff
@@ -124,7 +125,20 @@ abstract class SparkOperation(session: Session)
   override protected def setState(newState: OperationState): Unit = {
     super.setState(newState)
     if (eventEnabled) {
-      EventBus.post(SparkOperationEvent(this, operationListener.flatMap(_.getExecutionId)))
+      EventBus.post(SparkOperationEvent(
+        this,
+        operationListener.flatMap(_.getExecutionId),
+        operationListener.map(_.operationRunTime.get()),
+        operationListener.map(_.operationCpuTime.get())))
+      if (OperationState.isTerminal(newState)) {
+        operationListener.foreach(l => {
+          info(s"statementId=${statementId}, " +
+            s"operationRunTime=${KyuubiSparkUtil.formatDuration(l.operationRunTime.get())}, " +
+            s"operationCpuTime=${KyuubiSparkUtil.formatDurationNano(l.operationCpuTime.get())}")
```

Comment:
Use `org.apache.spark.ui.formatDurationVerbose` instead of introducing new methods.

Reply:
`UIUtils.formatDurationVerbose` is a non-public API, so I add SparkUIUtilsHelper to invoke it.